NightscoutServiceKit/RemoteCommands/V1/Notifications/RemoteNotification.swift

```diff
@@ -7,15 +7,19 @@
 //
 
```

Comment:
This is only used for Remote 1.0 (Loop today). It represents the payload for an incoming push notification.